conn.go

```diff
@@ -674,6 +680,14 @@ func (c *Conn) handleIncomingPacket(buf []byte, enqueue bool) (bool, *alert.Aler
 		var err error
 		buf, err = c.state.cipherSuite.Decrypt(buf)
 		if err != nil {
+			if len(c.state.SessionID) > 0 {
+				// According to the RFC, we need to delete the stored session.
+				// https://datatracker.ietf.org/doc/html/rfc5246#section-7.2
+				if err := c.fsm.cfg.sessionStore.Del(c.state.SessionID); err != nil {
```

Comment:
This err is intend to shadow the out one. Otherwise, the outer err will lost.
Should I use a new name like err1? @at-wat